src/client.ts

```diff
@@ -350,38 +369,21 @@ export class Strapi {
    * ```
    *
    * @see SingleTypeManager
-   * @see Strapi
+   * @see StrapiClient
    */
   single(resource: string) {
     return new SingleTypeManager(resource, this._httpClient);
   }
-
-  /**
-   * Returns access to the files API to interact with media files uploaded to Strapi.
-   *
-   * This property provides methods for retrieving files from the upload plugin: findOne, find.
-   *
-   * @returns An instance of {@link FilesManager} to work with files.
-   *
-   * @example
-   * ```typescript
-   * // Initialize the client with required configuration
-   * const client = new Strapi({ baseURL: 'http://localhost:1337/api' });
-   *
-   * // Example: retrieve a list of image files
-   * const imageFiles = await client.files.find({
-   *   filters: { mime: { $contains: 'image' } },
-   *   sort: 'name:asc'
-   * });
-   *
-   * // Example: retrieve a single file by ID
-   * const file = await client.files.findOne(1);
-   * ```
-   *
-   * @see FilesManager
-   * @see Strapi
-   */
-  get files() {
-    return this._filesManager;
-  }
 }
+
+// Support for deprecated types
+
+/**
+ * @deprecated This type will be removed in v2, consider using {@link StrapiClientConfig} as a replacement
+ */
+export type StrapiConfig = StrapiClientConfig;
+
+/**
+ * @deprecated This type will be removed in v2, consider using {@link StrapiClient} as a replacement
+ */
+export type Strapi = StrapiClient;
```

Comment:
I wasn't sure whether to export that from the client or handle deprecation and aliases in the root index.ts which represent the public API. Opinion wanted on this :)

Reply:
I have seen more and more a pattern of creating an exports.ts file with the public export and pointing to that instead of the index 🤷  to separate internal vs external. I think it make more sense to at least move it out of this file.

Reply:
Yeah, I had an api.ts file that contained all the public api, but reverted that to the regular index.ts in another PR to avoid creating useless noise, but I see how that could be useful now.

Might introduce this here to decouple things a bit

Reply:
Done in [fb2fd52](https://github.com/strapi/client/pull/73/commits/fb2fd5231877f3ba04897ad7ca261f274719731d)